coretex/entities/task_run/metrics/metric.py

```diff
@@ -68,6 +68,12 @@ def create(
                 range in which values will be displayed for y axis
         """
 
+        if isinstance(xType, int):
+            xType = createMetricType(xType)
```

Comment:
There is no need for a specific function for creating enum from type. You can just do `xType = MetricType(xType)`